tests/smoke.sh

```diff
@@ -125,89 +170,119 @@ function rpc_node_try {
-# 1406 config
-GRAPH_ADMIN_RPC_1406=9426
-GRAPH_IPFS_1406=6407
-OST_COGATEWAY_ADDRESS_1406=0x02cffaa1e06c28021fff6b36d9e418a97b3de2fc
+function test_goerli {
+    start_origin_node goerli geth
+    sleep 25
+    grep_try goerli geth
+    rpc_node_try "0005" # Given like this as it is used for the port in `rpc_node_try`.
+    deploy_subgraph_mosaic_config goerli 1405 origin $GRAPH_ADMIN_RPC_GOERLI $GRAPH_IPFS_GOERLI
+    sleep 25
+    rpc_origin_sub_graph_try $GRAPH_WS_PORT_GOERLI $OST_GATEWAY_ADDRESS_GOERLI_1405
+    # NOTE: as we currently do not publish token config on chain start, deployment fails.
+    # deploy_subgraph_gateway_config goerli 1405 origin $GRAPH_ADMIN_RPC_GOERLI $GRAPH_IPFS_GOERLI $WETH_GATEWAY_ADDRESS_GOERLI_1405
+    # sleep 25
+    # rpc_origin_sub_graph_try $GRAPH_WS_PORT_GOERLI $WETH_GATEWAY_ADDRESS_GOERLI_1405
+    stop_node goerli
+}
 
-# 1407 config
-GRAPH_ADMIN_RPC_1407=9427
-GRAPH_IPFS_1407=6408
-OST_COGATEWAY_ADDRESS_1407=0xf690624171fe06d02d2f4250bff17fe3b682ebd1
+function test_1406 {
+    start_auxiliary_node 1406 ropsten
+    sleep 25
+    grep_try 1406 geth
+    rpc_node_try 1406
+    deploy_subgraph_mosaic_config ropsten 1406 auxiliary $GRAPH_ADMIN_RPC_1406 $GRAPH_IPFS_1406
+    sleep 25
+    rpc_auxiliary_sub_graph_try 1406 $OST_COGATEWAY_ADDRESS_1406
+    stop_node 1406
+}
 
-# ropsten config
-GRAPH_ADMIN_RPC_ROPSTEN=8023
-GRAPH_IPFS_ROPSTEN=5004
-GRAPH_WS_PORT_ROPSTEN=60003
-OST_GATEWAY_ADDRESS_ROPSTEN_1406=0x04df90efbedf393361cdf498234af818da14f562
-OST_GATEWAY_ADDRESS_ROPSTEN_1407=0x31c8870c76390c5eb0d425799b5bd214a2600438
+function test_1407 {
+    start_auxiliary_node 1407 ropsten
+    sleep 25
+    grep_try 1407 geth
+    rpc_node_try 1407
+    deploy_subgraph_mosaic_config ropsten 1407 auxiliary $GRAPH_ADMIN_RPC_1407 $GRAPH_IPFS_1407
+    sleep 25
+    rpc_auxiliary_sub_graph_try 1407 $OST_COGATEWAY_ADDRESS_1407
+    stop_node 1407
+}
 
-# Dev chain config
-GRAPH_ADMIN_RPC_DEV_ORIGIN=9535
-GRAPH_IPFS_DEV_ORIGIN=6516
-GRAPH_WS_PORT_DEV_ORIGIN=61515
-OST_GATEWAY_ADDRESS_DEV_ORIGIN_WETH=0xaE02C7b1C324A8D94A564bC8d713Df89eae441fe
-OST_CO_GATEWAY_ADDRESS_DEV_ORIGIN_WETH=0xc6fF898ceBf631eFb58eEc7187E4c1f70AE8d943
+function test_1405 {
+    start_auxiliary_node 1405 goerli
+    sleep 25
+    grep_try 1405 geth
+    rpc_node_try 1405
+    deploy_subgraph_mosaic_config goerli 1405 auxiliary $GRAPH_ADMIN_RPC_1405 $GRAPH_IPFS_1405
+    sleep 25
+    rpc_auxiliary_sub_graph_try 1405 $OST_COGATEWAY_ADDRESS_1405
+    # NOTE: as we currently do not publish token config on chain start, deployment fails.
+    # deploy_subgraph_gateway_config goerli 1405 auxiliary $GRAPH_ADMIN_RPC_1405 $GRAPH_IPFS_1405 $WETH_GATEWAY_ADDRESS_GOERLI_1405
+    # sleep 25
+    # rpc_auxiliary_sub_graph_try 1405 $WETH_COGATEWAY_ADDRESS_1405
+    stop_node 1405
+}
 
-DEV_AUXILIARY_CHAIN_ID=1000
-GRAPH_ADMIN_RPC_DEV_AUXILIARY=9020
-GRAPH_IPFS_DEV_AUXILIARY=6001
+function test_dev_origin {
```

Comment:
We can move test_dev_origin along with test_ropsten and test_goerli for logical grouping for origin chains.